starboard/shared/win32/file_internal.cc

```diff
@@ -110,6 +110,10 @@ HANDLE OpenFileOrDirectory(const char* path, int flags) {
   }
 
   DWORD creation_disposition = 0;
+  if (!flags) {
+    creation_disposition = OPEN_EXISTING;
```

Comment:
why are we doing this?

Reply:
This correspond to http://shortn/_xC9YiezPWn this section. It was missing in previously.